### PR TITLE
[devtools] add keyboard navigation guide

### PR DESCRIPTION
**Pull request by robdodson**
What's changed, or what was fixed?
- Added navigating devtools with assistive technology guide.

I decided to keep it as a single guide for now because I think it may actually be easier to navigate instead of spreading it across multiple guides. @kaycebasques 	wdyt?

**Target Live Date:** 2019-03-14

I'm giving a talk at CSUN on 03/15 about the accessibility of devtools so I'd like to publish this guide before then if possible.

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

I noticed my use of `Warning:` did not seem to create the red aside. Not sure if that's a quirk of the staging environment? My use of `Note:` worked.

**CC:** @petele

### CODE REVIEW

**Comment by kaycebasques** [comment]
Single guide SGTM. I'm working on a fire drill right now but we can definitely land this before your talk.

**Comment by robdodson** [comment]
Rad, thank you Kayce :)

File: src/content/en/tools/chrome-devtools/accessibility/navigation.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+and the line number for the selector.
+
+#### Known issues
+
+- If you use the filter box you can no longer inspect styles.
+
+### Event listeners tab
+
+From within the **Elements** panel you may inspect the event listeners applied
+to an element using the **Event Listeners** tab. // TODO: Is this tab
```
Comment (robdodson) [inline src/content/en/tools/chrome-devtools/accessibility/navigation.md]: @kaycebasques 	left a todo. Is there an official doc that explains this tab anywhere? If so i was going to link to it.

File: src/content/en/tools/chrome-devtools/accessibility/navigation.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+  [the chromium tracking
+  bug](https://bugs.chromium.org/p/chromium/issues/detail?id=868480) for more
+  information.
+- The ARIA Attributes and Computed Properties sections are marked up as ARIA
+  trees, but they do not currently have focus managmenet so they are not
+  keyboard operable.
+
+## Audits panel
+
+The **Audits** panel let's you run a series of tests against a site to check
```
Comment (robdodson) [inline src/content/en/tools/chrome-devtools/accessibility/navigation.md]: @kaycebasques 	do you know if there is an official doc for the audits panel? If so i was going to link to it here.

**Pull request merged**